python/tvm/topi/adreno/conv2d_winograd_common.py

```diff
@@ -494,16 +494,17 @@ def schedule_conv2d_winograd(cfg, s, output, pre_computed):
         s[OL].set_scope("local")
         output = s.outputs[0]
 
-    m = alpha - 3 + 1
     if len(s[output].op.axis) == 4:
         n, co, h, w = s[output].op.axis
     else:
-        n, co, h, w, _ = s[output].op.axis
-    ho, wo, hi, wi = s[output].tile(h, w, m, m)
+        n, co, h, w, cb = s[output].op.axis
     inverse_scope, n = s[output].split(n, nparts=1)
 
-    fused = s[output].fuse(n, co, ho, wo)
+    fused = s[output].fuse(n, co, h, w)
     bb, tt = s[output].split(fused, 128)
+    if len(s[output].op.axis) > 4:
```

Comment:
I don;t like split of init cb and this verification but cannot propose how to do this better without code duplication

Reply:
Fixed. Replaced to check on `None`